lib/auth/desktop.go

```diff
@@ -28,6 +29,8 @@ import (
 	"github.com/gravitational/teleport/lib/tlsca"
 )
 
+var LicenseOID = asn1.ObjectIdentifier{1, 3, 1, 0, 0, 5}
```

Comment:
Should we move this to the same file as our other extensions to prevent future overlap?

Reply:
I moved it